keyboards/keychron/k3/rgb/v2/optical_iso/keymaps/default/keymap.c

```diff
@@ -0,0 +1,172 @@
+      KC_LSFT,   KC_NUBS,   KC_Z,      KC_X,    KC_C,    KC_V,    KC_B,     KC_N,    KC_M,    KC_COMM,   KC_DOT,     KC_SLSH,               KC_RSFT,   KC_UP,     KC_END    ,
+      KC_LCTL,   KC_LGUI,   KC_LALT,                              KC_SPC,                                KC_RALT,   MO(MAC_FN), KC_RCTL,    KC_LEFT,   KC_DOWN,   KC_RGHT
+  ),
+
+/*  Mac Fn overlay
+    +------------------------------------------------------------------------------+
+    |    | F1 | F2 | F3 | F4 | F5 | F6 | F7 | F8 | F9 | F10| F11| F12|BOOT|   |RMOD|
+    +------------------------------------------------------------------------------+
+    |    |    |    |    |    |    |    |    |    |    |    |    |    |        |RM_P|
+    +------------------------------------------------------------------------------+
+    |      |    |    |    |    |    |    |    |    |    |    |    |    |      |    |
+    +-------------------------------------------------------------------      -----+
+    |       |    |    |    |    |    |    |    |    |    |    |    |    |     |    |
+    +------------------------------------------------------------------------------+
+    |     |    |    |    |    |    |    |    |    |    |    |    |       |RSAI|    |
+    +------------------------------------------------------------------------------+
+    |     |     |     |                             |     |    |    |RHUD|RSAD|RHUI|
+    +------------------------------------------------------------------------------+
+*/
+  [MAC_FN] = LAYOUT_75_iso(
+  /*  0           1           2           3           4           5           6           7           8           9           10          11          12          13          14          15         */
+      _______,    KC_F1,      KC_F2,      KC_F3,      KC_F4,      KC_F5,      KC_F6,      KC_F7,      KC_F8,      KC_F9,      KC_F10,     KC_F11,     KC_F12,     QK_BOOT,    _______,    RGB_M_P    ,
+      _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,                _______    ,
+      _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,                            _______    ,
+      _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,                _______    ,
+      _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,                _______,    RGB_SAI,    _______    ,
+      _______,    _______,    _______,                                        _______,                                        _______,    _______,    _______,    RGB_HUD,    RGB_SAD,    RGB_HUI
```

Comment:
Many of the `RGB_*` keycode [have been deprecated since May](https://github.com/qmk/qmk_firmware/commit/224ff1d262402a48d1d04912be10d22074d78ced) so this would be a good time to update them.